### PR TITLE
Fix container image extension conflict hint

### LINKED ISSUE #13946: Error message about conflicting container image extensions is wrong
**Describe the bug**

When there are multiple container image extensions available, the user needs to select one, or the build fails.
An error message explains how the user is meant to select one of the available extensions.

However, the message displayed has a typo that can cause further confusion.
Also, it's not obvious where the property should be added.

**Expected behavior**
The message should mention `quarkus.container-image.builder`

**Actual behavior**
The message should mention `quarkus.container-image-builder` 
The last dash should be a dot instead.

### PR DESCRIPTION
**Pull request by iocanel**
Fixes: #13946

### CODE REVIEW

File: extensions/container-image/deployment/src/main/java/io/quarkus/container/image/deployment/ContainerImageCapabilitiesUtil.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ private ContainerImageCapabilitiesUtil() {
         if (activeContainerImageCapabilities.size() > 1) {
             throw new IllegalStateException(String.join(" and ", activeContainerImageCapabilities)
                     + " were detected, at most one container-image extension can be present.\n"
-                    + "Either remove the unneeded ones, or select one using the property 'quarkus.container-image-builder=<extension name (without the `container-image-` prefix)>'.");
+                    + "Either remove the unneeded ones, or select one adding the property 'quarkus.container-image.builder=<extension name (without the `container-image-` prefix)>' in application.properties or as a system property.");
```
Comment (geoand) [inline extensions/container-image/deployment/src/main/java/io/quarkus/container/image/deployment/ContainerImageCapabilitiesUtil.java:34]:
```suggestion
                    + "Either remove the unneeded ones, or select one by adding the property 'quarkus.container-image.builder=<extension name (without the `container-image-` prefix)>' in application.properties or as a system property.");
```

**Comment by geoand** [comment]
Sorry to be obnoxious, but could you squash the commits please?

**Comment by gsmet** [comment]
I wanted to do it but I can't push to your fork so you will have to do it yourself.

**Comment by geoand** [comment]
Yup, I tried that as well

**Comment by iocanel** [comment]
@geoand @gsmet: squashed commits. can you have a look now?

**Review by geoand — APPROVED** [review]

**Pull request merged**